### PR TITLE
Users Can Now Add & View Skill's Links

### LINKED ISSUE #13: As a user on a skill's page, I should be able to add a new link
1. Add a Modal for a adding a link
2. Confirm that Links are displayed: `Link_Type: Link_Name` and that it actually links to the correct URL

### PR DESCRIPTION
**Pull request by Hopding**
Fixes #13. Note that this PR relies on the backend having merged [this](https://github.com/maryvilledev/skilldirectory/pull/140) PR.

### CODE REVIEW

File: src/Skills.jsx [hunk 1]
```diff
@@ -26,34 +28,58 @@ class Skills extends Component {
...
 
-  openModal() {
-    this.setState({modalIsOpen: true});
+  openSkillModal() {
+    this.setState({skillModalIsOpen: true});
   }
-  openDeleteModal(){
-    this.setState({deleteModalIsOpen: true});
+
+  afterOpenSkillModal() {
```
Comment (thebho) [inline src/Skills.jsx]: Are these methods required?

**Review by thebho — COMMENTED** [review]
Looks good. Just curious about those stub methods.

File: src/Skills.jsx (hunk 1 above)
Comment (Hopding) [reply src/Skills.jsx]: They are not. Probably best to remove them.

**Comment by Hopding** [comment]
Changes made. Should be ready to merge 👍

**Review by thebho — APPROVED** [review]

**Review by dane-johnson — COMMENTED** [review]
LGTM

**Review by smokinGun — APPROVED** [review]

**Pull request merged**